vta/python/vta/exec/rpc_server.py

```diff
@@ -27,7 +27,7 @@ def server_start():
         os.path.abspath(os.path.expanduser(__file__)))
     proj_root = os.path.abspath(os.path.join(curr_path, "../../../../"))
     dll_path = find_libvta()[0]
-    cfg_path = os.path.abspath(os.path.join(proj_root, "build/vta_config.json"))
+    cfg_path = os.path.abspath(os.path.join(curr_path, "../../../config/vta_config.json"))
```

Comment:
can you set it to the old path, and when I fix the path, I'll set it to point to the config directory

Reply:
right now we should keep it consistent with the tutorial instructions. I agree with your view of having a single directory to read from, but I'll need to make the changes after your PR gets merged.